docs/docsite/rst/porting_guide_2.4.rst

```diff
@@ -0,0 +1,74 @@
+.. _porting_2.4_guide:
+
+*************************
+Ansible 2.3 Porting Guide
+*************************
+
+This section discusses the changes to how you use Ansible between Ansible 2.2 and Ansible 2.3.
+
+It is intended to assist in updating your playbooks, plugins, etc so they will work with this version of Ansible.
+
+
+We suggest you read this page along with https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#24-dancing-days---active-development before upgrading Ansible to understand what, if any updates you will need to make.
+
+This document is part of a collection on porting, the rest can be found at :ref:`porting_guides <Porting Guides>`.
```

Comment:
perod after "porting", then new sentence "The complete list of porting guides can be found..."